native/acter/src/api/push/notification_item.rs

```diff
@@ -926,552 +450,116 @@ impl NotificationItem {
 
         Ok(builder.build()?)
     }
+}
 
-    async fn for_acter_object(
+async fn convert_acter_model(client: Client, event: AnyActerEvent) -> Result<Activity> {
+    Ok(Activity::for_acter_model(client.store(), AnyActerModel::try_from(event)?).await?)
```

Comment:
Thanks to the new implementation, we can now have the model from the event and then directly turn that into an activity right here.